sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -863,6 +866,24 @@ synchronized void getLatestReleaseDetails(String distributionGroupId, String upd
         } else {
             url += String.format(GET_LATEST_PRIVATE_RELEASE_PATH_FORMAT, mAppSecret, releaseHash);
         }
+
+        /* Send install id once upon in-app update if feature flag is enabled */
+        String shouldUseInstallIdFeature = mContext.getString(R.string.install_id_feature_enabled);
```

Comment:
This is completely harmless to do that for everyone, why insisting on having a feature flag for this?

Reply:
This is still under construction, so we keep the whole thing under feature flags to be independent of release cycles etc. We can remove/enable it anytime. For now, I think that was a team decision, but we can reconsider this with @Meirav